algoliasearch/src/main/java/com/algolia/search/saas/BaseAPIClient.java

```diff
@@ -62,7 +62,9 @@
     private List<String> readHosts;
     private List<String> writeHosts;
 
-    /** HTTP headers that will be sent with every request. */
+    /**
+     * HTTP headers that will be sent with every request.
+     */
```

Comment:
Do you think the previous format was better? I'm used to reformatting code to the coding style before starting a refactoring, but I can do it in a separate commit instead (and eventually squash them before merge).

Regarding this PR, let me know if you want me to rebase and extract those format changes to a separate commit.